utils/actions/labelPullRequest.ts

```diff
@@ -0,0 +1,61 @@
+const { Configuration, OpenAIApi } = require("openai");
+import { Octokit } from "octokit";
+
+const configuration = new Configuration({
+  apiKey: process.env.OPEN_AI_KEY,
+});
+const openai = new OpenAIApi(configuration);
+export default function flagPullRequest({
+  prTitle, 
+  businessLogicSummary,
+  github_token,
+  owner,
+  repo,
+  issue_number
+}: {
+  prTitle?: string;
+  businessLogicSummary?: string;
+  github_token?: string;
+  owner: string;
+  repo: string;
+  issue_number: number
+}) {
+
+  if (!github_token) {
+    return { error: "no github token" };
+  }
+
+  const octokit = new Octokit({
```

Comment:
Should we check for missing params?
There is a function for that.

Reply:
Then why does every single file not have it? For example, getOpenAISummary.ts doesn't have the missingParams function imported. 

(To me, it's fine that the parent of all these files which is api/actions/github/route.ts handles the missingParams logic)

Reply:
But why do we check for **one** param? Not all? Is that the only one required?
```
  if (!github_token) {
    return { error: "no github token" };
  }
```

Reply:
This is validated by the parent function but I still added param validation